seacat/src/main/java/com/teskalabs/seacat/SeaCat.kt

```diff
@@ -4,17 +4,19 @@ import android.content.Context
 import android.util.Log
 import androidx.localbroadcastmanager.content.LocalBroadcastManager
 import java.net.Socket
+import java.security.KeyStore
 import java.security.Principal
 import java.security.PrivateKey
 import java.security.cert.CertificateFactory
 import java.security.cert.X509Certificate
 import java.util.concurrent.Callable
 import java.util.concurrent.Executors
 import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManager
+import javax.net.ssl.TrustManagerFactory
 import javax.net.ssl.X509KeyManager
-import javax.net.ssl.X509TrustManager
 
-class SeaCat(
+open class SeaCat(
```

Comment:
Needed?

Reply:
This is needed in case we want to allow overriding of `createTrustManagers()`
Do we?

Reply:
I would like to introduce an alternative way of this (e.g. SeaCat controller).